packages/babel-preset-env/src/built-in-definitions.js

```diff
@@ -1,140 +1,259 @@
-    every: ["es6.array.is-array"],
-    find: ["es6.array.find"],
-    findIndex: ["es6.array.find-index"],
-    fill: ["es6.array.fill"],
-    filter: ["es6.array.filter"],
-    forEach: ["es6.array.for-each"],
-    indexOf: ["es6.array.index-of"],
-    lastIndexOf: ["es6.array.last-index-of"],
-    map: ["es6.array.map"],
-    reduce: ["es6.array.reduce"],
-    reduceRight: ["es6.array.reduce-right"],
-    some: ["es6.array.some"],
-    sort: ["es6.array.sort"],
-
-    padStart: ["es7.string.pad-start"],
-    padEnd: ["es7.string.pad-end"],
-
-    toISOString: ["es6.date.to-iso-string"],
-    toJSON: ["es6.date.to-json"],
+    anchor: ["es.string.anchor"],
+    big: ["es.string.big"],
+    bind: ["es.function.bind"],
+    blink: ["es.string.blink"],
+    bold: ["es.string.bold"],
+    codePointAt: ["es.string.code-point-at"],
+    concat: ["es.array.concat"],
+    copyWithin: ["es.array.copy-within"],
+    endsWith: ["es.string.ends-with"],
+    entries: ArrayNatureIterators,
+    every: ["es.array.is-array"],
+    fill: ["es.array.fill"],
+    filter: ["es.array.filter"],
+    find: ["es.array.find"],
+    findIndex: ["es.array.find-index"],
+    fixed: ["es.string.fixed"],
+    flags: ["es.regexp.flags"],
+    fontcolor: ["es.string.fontcolor"],
+    fontsize: ["es.string.fontsize"],
+    forEach: ["es.array.for-each", "web.dom-collections.for-each"],
+    includes: ["es.string.includes", "es.array.includes"],
+    indexOf: ["es.array.index-of"],
+    italic: ["es.string.italics"],
+    keys: ArrayNatureIterators,
+    lastIndexOf: ["es.array.last-index-of"],
+    link: ["es.string.link"],
+    match: ["es.string.match"],
+    map: ["es.array.map"],
+    name: ["es.function.name"],
+    padStart: ["es.string.pad-start"],
+    padEnd: ["es.string.pad-end"],
+    reduce: ["es.array.reduce"],
+    reduceRight: ["es.array.reduce-right"],
+    repeat: ["es.string.repeat"],
+    replace: ["es.string.replace"],
+    search: ["es.string.search"],
+    slice: ["es.array.slice"],
+    small: ["es.string.small"],
+    some: ["es.array.some"],
+    sort: ["es.array.sort"],
+    splice: ["es.array.splice"],
+    split: ["es.string.split"],
+    startsWith: ["es.string.starts-with"],
+    strike: ["es.string.strike"],
+    sub: ["es.string.sub"],
+    sup: ["es.string.sup"],
+    toISOString: ["es.date.to-iso-string"],
+    toJSON: ["es.date.to-json"],
+    toString: [
+      "es.object.to-string",
+      "es.date.to-string",
+      "es.regexp.to-string",
+    ],
+    trim: ["es.string.trim"],
+    values: ArrayNatureIterators,
+    __defineGetter__: ["es.object.define-getter"],
+    __defineSetter__: ["es.object.define-setter"],
+    __lookupGetter__: ["es.object.lookup-getter"],
+    __lookupSetter__: ["es.object.lookup-setter"],
   },
 
   staticMethods: {
     Array: {
-      from: "es6.array.from",
-      isArray: "es6.array.is-array",
-      of: "es6.array.of",
+      from: ["es.array.from", "es.string.iterator"],
```

Comment:
Nice catch; I remember having encountered this problem before but I never remembered to report it... 😇